mkosi/resources/man/mkosi.1.md

```diff
@@ -624,7 +624,7 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
 `SplitArtifacts=`, `--split-artifacts=`
 :   The artifact types to split out of the final image. A comma-delimited
     list consisting of `uki`, `kernel`, `initrd`, `os-release`, `prcs`, `partitions`,
-    `roothash` and `tar`. When building a bootable image `kernel` and `initrd`
+    `metainfo`, `roothash` and `tar`. When building a bootable image `kernel` and `initrd`
```

Comment:
`metainfo` sounds a bit generic, this is only used by appstream, isn't it? Maybe this should be `appstream-metainfo`?